assets/src/components/d3/d3ViewsNarrative.js

```diff
@@ -0,0 +1,57 @@
+import { isInRange, isOutOfRange } from '../../util/chartsMisc'
+
+const resources = (resourceData, resourceType, gradeSelection, weekRange) => {
```

Comment:
Nit-pick: In my opinion, these functions (`resources`, `grades`) could be named better. I try to name functions with verbs (e.g. `createResourcesText`) because it helps me remember it's a function/method, though I understand in some cases they are used more like a property, which would make `resourcesText` okay. But when I see `resources` or `grades` alone (without the module name), it makes me think it is storing raw resource or grade data, which isn't true here. My two cents.